osbs/build/build_requestv2.py

```diff
@@ -436,6 +436,8 @@ def _update_trigger_imagestreamtag(self, source_registry):
         if image_name.registry == source_registry and self.organization:
             image_name.enclose(self.organization)
 
+        # we want to convert it only in registry, as there should be colon before tag
+        image_name.registry.replace(':', '-')
```

Comment:
Does this do anything? `replace` will not modify the string. It will create a new one which is not saved anywhere.

Reply:
fixed